make/photon/prepare/templates/registry/config.yml.jinja

```diff
@@ -9,7 +9,18 @@ storage:
   {{storage_provider_info}}
   maintenance:
     uploadpurging:
+{% if purge_upload.enabled %}
+      enabled: true
+      age: {{ purge_upload.age }}
+      interval: {{ purge_upload.interval }}
+      {% if purge_upload.dryrun %}
+      dryrun: true
+      {% else %}
```

Comment:
why not assign dryrun value like age and internal 
```
dryrun: {{ purge_upload.dryrun }}
```

Reply:
to avoid       
```
  dryrun: False
```